_topic_map.yml

```
@@ -34,6 +34,16 @@ Topics:
 - Name: Legal notice
   File: legal-notice
 ---
+Name: What's New?
+Dir: whats_new
+Distros: openshift-origin
+Topics:
+- Name: Overview
+  File: index
+ - Name: Release Notes
```

```suggestion
  - Name: Release Notes
```
A missing space breaks yaml structure